.evergreen/run-ocsp-test.sh

```diff
@@ -40,5 +40,5 @@ provision_ssl
 
 echo "Running OCSP tests with ${JDK}"
 ./gradlew -version
-./gradlew -PjdkHome=${JAVA_HOME} ${GRADLE_EXTRA_VARS} --stacktrace --debug --info driver-sync:test --tests OcspTest
+./gradlew -PjdkHome=/opt/java/${JDK} ${GRADLE_EXTRA_VARS} --stacktrace --debug --info driver-sync:test --tests OcspTest
```

Comment:
So we run Gradle with JDK 11, but run tests with JDK 17. I guess this is because of [_"Java 17 and later versions are not yet supported."_](https://docs.gradle.org/current/userguide/compatibility.html).

Reply:
That's right.  We have a separate issue for upgrading to Gradle 7.3.

Reply:
Or, we could do it with toolchains: https://docs.gradle.org/7.3-rc-3/userguide/toolchains.html